src/crimson/osd/osd_operations/client_request.cc

```diff
@@ -413,6 +413,11 @@ ClientRequest::do_process(
       DEBUGDPP("{}.{}: dropping misdirected op",
 	       *pg, *this, this_instance_id);
       co_return;
+    } else if (!pg->is_unreadable_object(m->get_hobj())) {
```

Comment:
Looking at the method name, should it be `if (pg->is_unreadable_object(m->get_hobj()))` to indicate missing?

Reply:
Yup.  Thanks for the catch.